### ISSUE TITLE
Is this Library really Done for? I don't think so.

### ISSUE THREAD

**Issue by LaserStony** [bluelinelabs/LoganSquare#227]
I don't think that LoganSquare is Done for... It still works Perfectly for me (up-till now) and there aren't any serious bugs either.. so I think, It's All good...

**Comment by ursusursus** [comment]
kotlin?

**Comment by simtel12** [comment]
@LaserStony Have you tried it with Gradle 5?  It's breaking for me.   https://github.com/simtel12/Gradle5LogansquareTest

**Comment by LaserStony** [comment]
@simtel12 Nope. I haven't checked it with Gradle 5 yet... Will check it out And report ASAP..

**Comment by simtel12** [comment]
> @simtel12 Nope. I haven't checked it with Gradle 5 yet... Will check it out And report ASAP..

@LaserStony  Quick update: I found out that for some reason the generated classes are not being compiled with the transient dependency on jackson-core.  If you manually add an implementation dependency on jackson-core, then it works just fine!

**Comment by LaserStony** [comment]
@simtel12 Ooo.. Nice!! Thanks for the Heads up...
